UI/Web/src/styles.scss

```diff
@@ -70,6 +70,10 @@ app-root {
   scrollbar-color: var(--primary-color-scrollbar);
 }
 
+.justify-content-between, .modal-title {
```

Comment:
Don't add styles to the global css. This needs to be copied to the places where it's used.

Reply:
Oh! I've just read this recommendation...
So with this fix we deal with the issue more broadly without having to add the style to all places individually. It shouldn't break anything else because it's just 'breaking' long strings as needed...
If you say it's not the way to go, I'll look into some other fix.

Reply:
Applying on .justify-content-between has unintended consequences. Please duplicate the style for now.